driver/src/main.rs

```diff
@@ -165,14 +166,11 @@ struct Options {
     )]
     earliest_solution_submit_time: Duration,
 
-    /// Subsidy factor used to compute the minimum average fee per order in a
-    /// solution as well as the gas cap for economically viable solution.
-    #[structopt(
-        long,
-        env = "ECONOMIC_VIABILITY_SUBSIDY_FACTOR",
-        default_value = "10.0"
-    )]
-    economic_viability_subsidy_factor: f64,
+    /// Subsidy factor used to compute the minimum average fee per order in a solution as well as
+    /// the gas cap for economically viable solution.
+    /// If not set then only default_min_avg_fee_per_order and default_max_gas_price are used.
```

Comment:
I find the default value for this unintuitive. To me not specifying a subsidy factor is equivalent to not subsidizing, i.e. `1.0`. Maybe this should be a separate flag (although that is also not super nice because we have enough CLI flags as it is).

Reply:
@nlordell implemented it by introducing a new enum EconomicViabilityStrategy.